KalebIrwin-Power-signoff.md

```diff
@@ -0,0 +1,42 @@
+## Function of the subsystem 
+The subsystem will provide the microcontroller and sensor interface with power. There will be two configurations designed. The first configuration will be designed to operate with input from an electrical outlet and the second configuration will be powered by a battery.
+
+## Constraints 
```

Comment:
List voltages, voltage tolerances, maximum current, average current or power for each item this subsystem is powering.

Put a space between numbers and units ("12 V" instead of "12V").

Amp-hours are not a measure of power. Be careful about separating current computations and power computations (in this entire document, not just this part).

Need to specify how long it can run on battery without recharging. You sort-of do this but be more explicit about listing it as a system requirement.